src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -90,20 +88,34 @@ export class CreateScenariosComponent {
   @Input() plan$ = new BehaviorSubject<Plan | null>(null);
   @Input() planningStep: PlanStep = PlanStep.CreateScenarios;
   @Output() changeConditionEvent = new EventEmitter<string>();
-  @Output() changeConstraintsEvent = new EventEmitter<Constraints>();
-  @Output() changePrioritiesEvent = new EventEmitter<Priorities>();
 
   formGroups: FormGroup[];
   readonly PlanStep = PlanStep;
   panelExpanded: boolean = true;
   stepStates: StepState[];
 
   constructor(private fb: FormBuilder) {
+    // TODO: Get and populate saved scenario config
+
     this.formGroups = [
       // Step 1: Select condition score
       this.fb.group({
         scoreSelectCtrl: ['', Validators.required],
       }),
+      // Step 2: Set constraints
+      this.fb.group({
+        budgetForm: this.fb.group({
+          maxBudget: [''],
+          optimizeBudget: [false, Validators.required],
```

Comment:
We can add more validation on the form inputs later (like invalid numbers, max, etc)